File: CSX/Functions and Execution Context/findWaldo.js
function findWaldo(obj) {
    for (let entry of Object.entries(obj)) {
        if (entry[0] == 'Waldo') return entry[1];
    }
    return "Where's Waldo?"
}

const DC = {'Bruce': 'Wayne', 'Harley': 'Quinn'}
const supernatural = {'Sam': 'Winchester', 'Dean': 'Winchester', 'Waldo': 'unknown'}
console.log(findWaldo(DC))
console.log(findWaldo(supernatural))

File: CSX - Python/Functions and Execution Context/find_waldo.py
def find_waldo(locations):
    if 'Waldo' in locations:
        return locations['Waldo']
    return "Where's Waldo?"

dc = { 'Bruce': 'Wayne', 'Harley': 'Quinn' }
supernatural = { 'Sam': 'Winchester', 'Dean': 'Winchester', 'Waldo': 'unknown', }

print(find_waldo(dc))
print(find_waldo(supernatural))
